src/components/br.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function P(){
+    return <p>hello world</p>
+}
+
+export default P;
```

Comment:
New newline at end of file. Make sure your team uses tools like `prettier` to format your code properly.